types.go

```diff
@@ -23,6 +23,7 @@ type User struct {
 	GivenName       string `bson:"given_name"`
 	FamilyName      string `bson:"family_name"`
 	IsRegistered    bool   `bson:"is_registered"`
+	Password        string `bson:"password"`
```

Comment:
I think you can remove the bson annotations

Reply:
Agreed. That was for Mongo.